pkg/vault/contracts/authorizer/TimelockAuthorizer.sol

```diff
@@ -737,7 +801,14 @@ contract TimelockAuthorizer is IAuthorizer, IAuthentication, ReentrancyGuard {
         bytes memory data = abi.encodeWithSelector(this.grantPermissions.selector, _ar(actionId), account, _ar(where));
 
         // TODO: fix actionId for event (maybe overhaul _scheduleWithDelay?)
-        return _scheduleWithDelay(0x0, address(this), data, delay, executors);
+        uint256 id = _scheduleWithDelay(0x0, address(this), data, delay, executors);
+        // accounts that schedule actions are automatically made cancelers for them, so that they can manage their
```

Comment:
```suggestion
        // Granters that schedule actions are automatically made cancelers for them, so that they can manage their
        // action. We check that they are not already a canceler since e.g. root may schedule grants (and root is
        // always a global canceler).
```